planning/behavior_velocity_planner/src/scene_module/crosswalk/scene_walkway.cpp

```diff
@@ -80,18 +81,21 @@ bool WalkwayModule::modifyPathVelocity(
     stop_factor.stop_factor_points.emplace_back(debug_data_.nearest_collision_point);
     planning_utils::appendStopReason(stop_factor, stop_reason);
 
-    // update state
-    const Point self_pose = {
-      planner_data_->current_pose.pose.position.x, planner_data_->current_pose.pose.position.y};
-    const Point stop_pose = {
-      debug_data_.first_stop_pose.position.x, debug_data_.first_stop_pose.position.y};
-    const double distance = bg::distance(stop_pose, self_pose);
+    // use arc length to identify if ego vehicle is in front of walkway stop or not.
+    const double distance = tier4_autoware_utils::calcSignedArcLength(
```

Comment:
pls change to signed_arc_dist_to_stop_point

Reply:
fixed at 76c7ce9